extras/paid-books.md

```diff
@@ -91,6 +91,11 @@ Name | Author | ISBN
 ### Cloud Computing
 
 ### Math (Linear Algebra)
+Name | Author | ISBN
+:-- | :--: | :--:
+[Linear Algebra Done Right third edition](http://www.amazon.in/Linear-Algebra-Right-Undergraduate-Mathematics/dp/0387982582) | Sheldon Axler | 0387982582
+
```

Comment:
No worries at all! Unfortunately I can't merge the pull request, you'll have to wait for Eric to do that, but thanks for your submissions!